modules/claims_api/spec/requests/v2/veterans/rswag_disability_compensation_request_spec.rb

```diff
@@ -696,7 +837,7 @@ def make_request(example)
       describe 'Getting a 404 response' do
         response '404', 'Resource not found' do
           schema JSON.parse(Rails.root.join('spec', 'support', 'schemas', 'claims_api', 'v2', 'errors',
-                                            'disability_compensation', 'default.json').read)
+                                            'disability_compensation', 'default_without_source.json').read)
 
```

Comment:
@rockwellwindsor-va I am probably missing something, but I am not sure why the attachments endpoint has a different 404 response than the other endpoints.
![Screenshot 2024-05-30 at 12 17 28 PM](https://github.com/department-of-veterans-affairs/vets-api/assets/25069483/fc6f50d2-2d49-43cd-8369-56a6bcb1b8e1)

Reply:
@stiehlrod That was actually already in there.  This endpoint does not build and validate the target veteran so there is no update for this endpoint on this one.